exim/datadog_checks/exim/check.py

```diff
@@ -0,0 +1,77 @@
+import re
+from collections import namedtuple
+from typing import Any
+
+from datadog_checks.base import AgentCheck
+from datadog_checks.base.utils.subprocess_output import get_subprocess_output
+
+
+class EximCheck(AgentCheck):
+
+    # This will be the prefix of every metric and service check the integration sends
+    __NAMESPACE__ = 'exim'
+
+    SERVICE_CHECK_NAME = 'returns.output'
+
+    def __init__(self, name, init_config, instances):
+        super(EximCheck, self).__init__(name, init_config, instances)
+
+    def check(self, instance):
```

Comment:
```suggestion
    def check(self, _):
```
this parameter is deprecated. You can access the instance at any point with `self.instance`